app/assets/javascripts/discourse/tests/unit/models/extra-nav-item-test.js

```diff
@@ -0,0 +1,25 @@
+import { setupTest } from "ember-qunit";
+import { module, test } from "qunit";
+import { ExtraNavItem } from "discourse/models/nav-item";
+
+module("Unit | Model | extra-nav-item", function (hooks) {
+  setupTest(hooks);
+
+  test("displayName updates when count property changes", function (assert) {
+    const extraNavItem = ExtraNavItem.create({
+      name: "something",
+    });
+
+    assert.strictEqual(
+      extraNavItem.displayName,
+      "[en.filters.something.title count=0]"
+    );
+
+    extraNavItem.count = 2;
+
+    assert.strictEqual(
```

Comment:
Pretty sure this test would pass even before the fix, right? Because we're not really testing the autotracking here, we're just re-running the `displayName` getter (which has no caching) 🤔 

Fix itself looks good though, thanks!